### DISCUSSION TITLE
select_taxa.py only recognize short names

### DISCUSSION THREAD

**Discussion by achoubc** [TheBrownLab/PhyloFisher#58] [Q&A] [answered]
Hello, 

"select_taxa.py" with the "--to_exclude" option seem to only recognize the abbreviated names (strangely, except Telonemida). 
Would it be possible to make it recognize by the group name or by different classification as in the older version please?

Thank you again for your time!
Best, 

Anna

**Comment by robert-ervin-jones** [comment, marked as answer]
Hi Anna,

Thanks for reaching out. This is actually a bug. I have opened an issue and am working on fixing it now. 

Best,
Robert
